tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Services/CreateOrganisationServiceTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Identity.Common.Models;
+using NHSD.BuyingCatalogue.Identity.Common.Results;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+using NHSD.BuyingCatalogue.Organisations.Api.Repositories;
+using NHSD.BuyingCatalogue.Organisations.Api.Services;
+using NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Builders;
+using NHSD.BuyingCatalogue.Organisations.Api.Validators;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Services
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public sealed class CreateOrganisationServiceTests
+    {
+        [Test]
+        public void Constructor_Null_Repository_Throws()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                var _ = new CreateOrganisationService(null, Mock.Of<IOrganisationValidator>());
+            });
+        }
+
+        [Test]
+        public void Constructor_Null_Validator_Throws()
```

Comment:
I think `_NullValidator_` would be clearer.

Reply:
👍